app/views/users/show.html.haml

```diff
@@ -0,0 +1,9 @@
+- content_for(:page_title) do
+  %h1 Welcome to #{@user.name}'s profile!
```

Comment:
Can you combine this with the user's posts index?  It seems to me that the user's profile and their posts list should be the same page.